File: Sort/Select Sort/select_sort.py
def select_sort(array):    
    for i in range(len(array)):        
        index = i
        for j in range(i + 1,len(array)):
            if array[index] > array[j]:
                index = j                
        array[i],array[index] = array[index],array[i]   
    return array
    


    
    
    
assert select_sort( [18, 16, 13, 19, 15, 14, 17] ) ==  [13, 14, 15, 16, 17, 18, 19] 

File: Sort/Select Sort/select_sort.cs
class Sort
{
    public int[] select_sort(int[] arr)
    {
        int index = 0 ;
        int temp = 0;
        for (int i = 0; i < arr.Length - 1; i++)
        {
            index = i;
            for (int j = i + 1; j < arr.Length ; j++)
            {
                if (arr[j] < arr[index])
                {
                    index = j;                   
                }
            }
            if (index != i)
            {
                temp = arr[index];
                arr[index] = arr[i];
                arr[i] = temp;
            }
            
        }
        return arr;
    }       
}